examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -8,8 +8,14 @@ authToken: ""
 
 # Address of the teleport proxy with port (usually :3080).
 proxyAddr: ""
-# Comma-separated list of roles to enable (any of: kube,db,app)
-roles: "kube"
+# Comma-separated list of roles to enable (any of: kube,db,app,discovery)
+roles: "kube,app,discovery"
+
+# enableDiscovery controls whether we enable discovery service.
+# It is used to handle backward compatibility with version that didn't
+# have discovery and their tokens didn't allow for `discovery` or `app`
+# services to run
+enableDiscovery:
```

Comment:
move this into  `kubernetesDiscovery` block so everything can be placed under the same block